apps/bmd/src/pages/PollWorkerScreen.tsx

```diff
@@ -330,29 +466,70 @@ const PollWorkerScreen: React.FC<Props> = ({
             }
           />
         )}
+        {isSavingTally && (
+          <Modal content={<Loading>Saving to pollworker card…</Loading>} />
```

Comment:
Either just "card" or "Poll Worker card" or "poll worker card" or something like that.
```suggestion
          <Modal content={<Loading>Saving to card…</Loading>} />
```